docs/topics/class-based-views/intro.txt

```diff
@@ -286,10 +286,30 @@ of the method to be decorated as the keyword argument ``name``::
     class ProtectedView(TemplateView):
         template_name = 'secret.html'
 
+Whenever you have a set of common decorators used redundantly you can define
+an iterable of decorators, i.e. a tuple or a list, and use this instead of invoke
+``method_decorator()`` multiple times::
+
+    decorators = (
+        never_cache,
+        login_required
+    )
+
+    @method_decorator(decorators, name='dispatch')
+    class ProtectedView(TemplateView):
+        template_name = 'secret.html'
+
+Remember that the ``request`` object is managed using a last-in-first-out basis,
```

Comment:
In my opinion, it would be more intuitive if the list was reversed internally so that decorators are run in the order they appear in the list.